### PR TITLE
Added hw23 solution

### PR DESCRIPTION
**Pull request by IvanKrivoruchkoHub**

### CODE REVIEW

File: src/main/java/core/basesyntax/MainApp.java [hunk 1]
```diff
@@ -12,5 +12,15 @@
  * - Отправить все машини на работу и остановить работу всех машин.
  */
 public class MainApp {
-
+    public static void main(String[] args) {
+        Bulldozer bulldozer = new Bulldozer();
```
Comment (Sofasmile) [inline src/main/java/core/basesyntax/MainApp.java]: What will happen if we write:
Machine bulldozer = new Bulldozer();
and the same with other objects?

Comment (IvanKrivoruchkoHub) [reply src/main/java/core/basesyntax/MainApp.java]: the result will not change

Comment (Sofasmile) [reply src/main/java/core/basesyntax/MainApp.java]: What will happen if we write:
Machine bulldozer = new Bulldozer();
and the same with other objects?

Comment (Sofasmile) [reply src/main/java/core/basesyntax/MainApp.java]: How do you think, why do we create abstract class Machine?
And what us the difference using Bulldozer or Machine?

**Comment by IvanKrivoruchkoHub** [comment]
changed the child class to parent when objects were are declared

File: src/main/java/core/basesyntax/MainApp.java (hunk 1 above)
Comment (IvanKrivoruchkoHub) [reply src/main/java/core/basesyntax/MainApp.java]: In Machine  we add the methods that we need to inplement in child classes.
We can write any Machine subclass object in the Machine class object.

**Pull request closed without merging**